lib/PlacesFeed.jsm

```diff
@@ -283,34 +283,69 @@ class PlacesFeed {
     _target.browser.ownerGlobal.gURLBar.search(`${data.label} `);
   }
 
+  _getSearchAlias() {
```

Comment:
I'll change the name to `_getSearchPrefix()` because otherwise I have to add another condition below to add the space only `if (searchAlias.length > 0) {}`.